pocs/observatory.py

```diff
@@ -70,16 +77,25 @@ def __init__(self, *args, **kwargs):
 # Properties
 ##########################################################################
 
-    @property
-    def is_dark(self):
-        horizon = self.location.get('twilight_horizon', -18 * u.degree)
+    def is_dark(self, horizon='astro'):
+
+        try:
+            horizon_deg = self.config['location']['{}_horizon'.format(horizon)]
+        except KeyError:
+            # Look for legacy name. Note this will happen for either horizon
+            try:
+                horizon_deg = self.config['location']['twilight_horizon']
+            except KeyError:
+                self.logger.info("Can't find twilight_horizon, using -18°")
+                horizon_deg = -18 * u.degree
 
         t0 = current_time()
-        is_dark = self.observer.is_night(t0, horizon=horizon)
+        is_dark = self.observer.is_night(t0, horizon=horizon_deg)
 
         if not is_dark:
             sun_pos = self.observer.altaz(t0, target=get_sun(t0)).alt
-            self.logger.debug("Sun {:.02f} > {}".format(sun_pos, horizon))
+            self.logger.debug("Sun {:.02f} > {} [{}]".format(
```

Comment:
Perhaps swap the order of horizon and horizon_deg; e.g.:

Sun -10 > -18 ("astro" horizon)